src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -138,7 +138,7 @@ public class TypicalPersons {
     private TypicalPersons() {} // prevents instantiation
 
     private static String getNextUniqueId() {
-        return String.format("#%06d", uniqueIdCounter++);
+        return String.format("%06d", uniqueIdCounter++);
```

Comment:
I suspect this is why JsonAddressBookStorageTest > readAndSaveAddressBook_allInOrder_success() FAILED